### ISSUE TITLE
color attribute for FloatingActionButton conflicts with an attribute defined in android.support.v7.appcompat

### ISSUE THREAD

**Issue by bengvr3** [FaizMalkani/FloatingActionButton#26]
I suggest renaming color to colorBackground.

**Comment by Aimen7** [comment]
Just rename the color attribute to whatever else, alse make sure to rename the FloatingActionButton_color in FloatingActionButton class. That was my solution.

**Comment by bengvr3** [comment]
That fixes the problem. You might want to consider changing it in the
future.

**Comment by bytehala** [comment]
Agreed with bvr, every other person using the library shouldn't have to always change it in their own copy. FAB as a library right now doesn't play nice with the compatibility lib.
